db/migrate/20160618180208_create_social_workers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSocialWorkers < ActiveRecord::Migration
+  def change
+    create_table :social_workers do |t|
+      t.string :last_name
+      t.string :first_name
+      t.string :email
+      t.string :phone
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
If social workers are currently identified by ids, should we capture and store that as well?